packages/current/current.0.6/opam

```diff
@@ -0,0 +1,56 @@
+opam-version: "2.0"
+synopsis: "Pipeline language for keeping things up-to-date"
+description: """\
+OCurrent provides an OCaml eDSL for writing CI/CD pipelines.
+
+It is used in ocaml-ci (which provides CI for OCaml projects on GitHub),
+and in docker-base-images (a pipeline that builds Docker images for various
+Linux distributions, OCaml compiler versions and CPU types, and pushes them
+to Docker Hub).
+
+A pipeline is written much like you would write a one-shot sequential script,
+but OCurrent will automatically re-run steps when the inputs change, and will
+run steps in parallel where possible."""
+maintainer: "talex5@gmail.com"
+authors: "talex5@gmail.com"
+homepage: "https://github.com/ocurrent/ocurrent"
+doc: "https://ocurrent.github.io/ocurrent/"
+bug-reports: "https://github.com/ocurrent/ocurrent/issues"
+depends: [
+  "ocaml" {>= "4.08.0"}
+  "current_incr" {>= "0.5"}
+  "fmt" {>= "0.8.9"}
+  "bos"
+  "ppx_deriving"
+  "lwt" {>= "4.3.0"}
+  "cmdliner" {>= "1.1.0"}
+  "sqlite3"
+  "duration"
+  "prometheus"
+  "dune" {>= "2.0"}
```

Comment:
For the revdeps errors, should I be removing the explicit dependency on `dune` or increasing the version number to 2.9 something?

Reply:
Why is dune.2.8.0 picked by the CI instead of 3.0?

Reply:
```suggestion
  "dune" {>= "2.9"}
```

Reply:
> Why is dune.2.8.0 picked by the CI instead of 3.0?

Because it’s also testing the lower-bound constraints (oldest possible versions of all the dependencies)